### PR TITLE
Add public setOptions method

### PR DESCRIPTION
**Pull request by AntonChesnokov**

### CODE REVIEW

**Comment by jeffchan** [comment]
@Tuizi can you take a look?

File: README.md [hunk 1]
```diff
@@ -22,6 +22,9 @@ Truncate.js currently depends on jQuery. There are two ways to use Truncate.js:
     // Redo the truncation (uses cached HTML).
     $('#truncate_me').truncate('collapse');
 
+    // Update options.
+    $('#truncate_me').truncate('setOptions', { lines : 3 });
```
Comment (Tuizi) [inline README.md]: could it be `options` instead of `setOptions`?
`$('#truncate_me').truncate('options', { lines : 3 });`

**Comment by AntonChesnokov** [comment]
>could it be options instead of setOptions?
$('#truncate_me').truncate('options', { lines : 3 });

Yes, no problem.

**Comment by AntonChesnokov** [comment]
@Tuizi @jeffchan feedback?

**Comment by Tuizi** [comment]
Oh sorry @AntonChesnokov I didn't saw your previous comment. Let me check something, because if "options" already exists what is the purpose of setOptions?

**Comment by AntonChesnokov** [comment]
@Tuizi `config` property or public method?

**Comment by Tuizi** [comment]
Exactly what you did, just rename `setOptions` method by `config`

**Pull request merged**